linebot/models/flex_message.py

```diff
@@ -501,7 +501,7 @@ def __init__(self,
                  aspect_mode=None,
                  background_color=None,
                  action=None,
-                 animated=None,
+                 animated=False,
```

Comment:
I think `None` is better than `False` because key with `None` value will be ignored in json result.

https://github.com/line/line-bot-sdk-python/blob/f520f9c80382367ace78322ffc8d99cb9bd99f26/linebot/models/base.py#L84